README.md

```diff
@@ -42,8 +42,29 @@ Further, DoDiscover seeks to help the user feel confident with their assumptions
 
 ## Unite causal discovery and causal representation learning
 
-Causal representation learning is the task of learning high-level latent variables and the causal structure between them from low-level variables observed in data.
-DoDiscover seeks to support causal representation learning algorithms in the context of traditional causal discovery settings.
+DoDiscover is a Python library for causal discovery (causal structure learning).
+Our goal is to provide developers and researchers with guide rails for causal discovery that doesn't require deep knowledge of individual causal discovery algorithms.
+
+## What is the difference between dodiscover and causal-learn?
+
+The goal of dodiscover is to flatten the on-ramp to causal discovery algorithms.
+DoWhy provides a consistent API for various method for causal effect estimation.
+DoDiscover aims to do the same with causal discovery algorithms.
```

Comment:
Accordingly: "DoDiscover aims to provide a cohesive and user-friendly API to apply causal discovery algorithms for inferring a causal graph from data."

Reply:
[db3001a](https://github.com/py-why/dodiscover/pull/132/commits/db3001a882929281d1710856a90e539c8112e52d)